android/src/main/java/com/reactnativemembrane/RNEndpoint.kt

```diff
@@ -1,17 +1,20 @@
 package com.reactnativemembrane
 
+import org.membraneframework.rtc.events.TracksAdded
 import org.membraneframework.rtc.media.AudioTrack
 import org.membraneframework.rtc.media.VideoTrack
 import org.membraneframework.rtc.utils.Metadata
 
-data class RNEndpoint (
-    val id: String,
-    val metadata: Metadata? = hashMapOf(),
-    val type: String?,
-    val videoTracks: HashMap<String, VideoTrack> = hashMapOf(),
-    val audioTracks: HashMap<String, AudioTrack> = hashMapOf(),
-    val tracksMetadata: HashMap<String, Metadata> = hashMapOf()
-) {
+data class RNEndpoint(
+  val id: String,
+  val metadata: Metadata? = hashMapOf(),
+  val type: String?,
+  val videoTracks: HashMap<String, VideoTrack> = hashMapOf(),
+  val audioTracks: HashMap<String, AudioTrack> = hashMapOf(),
+  val tracksMetadata: HashMap<String, Metadata> = hashMapOf(),
+  val tracks: Map<String, TracksAdded.Data.TrackData> = mapOf(),
```

Comment:
why are there videoTracks, audioTracks and tracks? o.o are they yet another tracks? ;)

Reply:
I do not know, it was before, i just adjusted this so it can contain simulcast data